### PR TITLE
Add QAZ keyboard

### PR DESCRIPTION
**Pull request by tominabox1**
Adding QAZ keyboard to VIA

## Checklist

- [x] The QMK source code follows the guide here: https://caniusevia.com/docs/configuring_qmk
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by Wilba6582** [comment]
This needs a QMK PR.

**Comment by tominabox1** [comment]
@Wilba6582 its already in the master branch

File: src/qaz/qaz.json [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        14,
+        15,
+        16,
+        17,
+        18,
+        19,
+        20,
+        21
+     ]
+  },
```
Comment (Wilba6582) [inline src/qaz/qaz.json:72]: this can just be `  "lighting": "qmk_rgblight",`

Comment (tominabox1) [reply src/qaz/qaz.json]: Fixed and tested.

**Pull request merged**